library/oneview_volume_facts.py

```diff
@@ -50,53 +39,63 @@
 EXAMPLES = '''
 - name: Gather facts about all Volumes
   oneview_volume_facts:
-    config: "{{ config_path }}"
-
+    hostname: 172.16.101.48
+    username: administrator
+    password: my_password
+    api_version: 500
 - debug: var=storage_volumes
 
 - name: Gather paginated, filtered and sorted facts about Volumes
   oneview_volume_facts:
-    config: "{{ config }}"
+    hostname: 172.16.101.48
+    username: administrator
+    password: my_password
+    api_version: 500
     params:
       start: 0
       count: 2
       sort: 'name:descending'
       filter: "provisionType='Thin'"
-
 - debug: var=storage_volumes
 
 - name: "Gather facts about all Volumes, the attachable volumes managed by the appliance and the extra managed
          storage volume paths"
   oneview_volume_facts:
-    config: "{{ config_path }}"
```

Comment:
Do we need to replace config with the actual parameter values.

Reply:
It is not a *must* in our own repo, but Ansible firmly requested it in several PRs, so it was updated when integrating into Ansible Core. 
It should be a best practice overall to keep our documentation synced with the modules in the ansible core repository.

Reply:
It is not a must in our own repo, but Ansible firmly requested it in several PRs, so it was updated when integrating into Ansible Core.
It should be a best practice overall to keep our documentation synced with the modules in the ansible core repository.